third_party/statusor/BUILD

```diff
@@ -0,0 +1,24 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "statusor_lib",
+    srcs = [
+        "statusor.cc",
+    ],
+    hdrs = [
+        "statusor.h",
+        "statusor_internals.h",
+    ],
+    visibility = ["//visibility:public"],
+    deps = ["@com_google_absl//absl/status"]
+    #deps = ["//source/common/common:assert_lib"],
+)
+
+# TODO (Michelle): fix tests
```

Comment:
Create a GitHub issue for tasks that needs to be done later. Then reference those: e.g. TODO(#X): fix test

Reply:
Just did the TODO instead :)